index.js

```diff
@@ -40,9 +40,10 @@ module.exports = function (target, dest, opts) {
     var fetchArgs = opts.link ? ['link'] : ['install'];
     opts = opts || {};
     var tree1;
+    opts.production = opts.production || true;
```

Comment:
Actually I think this logic needs to be updated.

If `opts.production` is false, you get `opts.production = false || true` which means the user setting `opts.production` to `false` is ignored.

I think a better solution is 
```
opts.production = opts.production ? opts.production : true;
```
This would check to see if `opts.production` exists (is defined). If it is defined, use that value, if it is undefined, use `true`

Would be worth adding a test where `opts.production` is `false` and confirming `--production` is not added to `fetchArgs`